src/js/components/status/LiveStatusIndicator.jsx

```diff
@@ -1,25 +1,8 @@
-// @flow
-
 import React, {Component, PropTypes} from 'react';
 import moment from 'moment';
 import { StatusIndicator, decodeResultValue } from './StatusIndicator';
 
-const {number, string} = PropTypes;
-
-// TODO: avoid duplicating flow constants from StatusIndicator
-const validResultValues = {
-    success: 'success',
-    failure: 'failure',
-    running: 'running',
-    queued: 'queued',
-    unstable: 'unstable',
-    aborted: 'aborted',
-    not_built: 'not_built',
-    unknown: 'unknown'
-};
-
-// Enum type from const validResultValues
-export type Result = $Keys<typeof validResultValues>;
+const {bool, number, string} = PropTypes;
```

Comment:
Yup. Otherwise bad things will almost certainly happen, as `React.PropTypes.*` are run-time values (and functions), which can be used as types in Flow. Either when they're constructor functions, or for enum unions; a Flow union could be `string | number` being flow types, or it might be `-1 | 0 | 1` being JS values